amqp/src/main/scala/akka/stream/contrib/amqp/model.scala

```diff
@@ -4,6 +4,7 @@
 package akka.stream.contrib.amqp
 
 import scala.collection.immutable.Seq
```

Comment:
I'd rather use
```
import scala.collection.immutable
```

```
immutable.Seq
```

because it's easy that such imports are accidentally lost (by tools, or merges)